packages/govern-console/src/utils/urls.tsx

```diff
@@ -8,3 +8,16 @@
 export function settingsUrl(daoName: string): string {
   return `/daos/${daoName}/dao-settings`;
 }
+
+/**
+ * Returns the url of the Proposal details page
+ * @param daoName the name of the DAO
+ * @param containerHash the proposal container hash
+ * @returns {string} the url of the proposal details page
+ */
+export function proposalDetailsUrl(
+  daoName: string,
+  containerHash: string,
+): string {
+  return `/proposals/${daoName}/${containerHash}`;
```

Comment:
In a perfect world, the url `/proposals/${daoName}/${containerHash}` could be changed to `${daoName}/proposals//${containerHash}`, but this will need the route change too. You can decide if you wanna do it .

Reply:
I will create a task to track the work for this.. trying to keep the changes in this PR relevant to this task only.